lessons/Algebra2/exploring-quadratic-models/langs/en-us/pages/exploring-mpg.adoc

```diff
@@ -0,0 +1,81 @@
+= Exploring the Fuel Efficiency Dataset
+++++
+<style>
+body.workbookpage td .autonum::after { content: ')' !important; }
+</style>
+++++
+For this page, you'll need to the  @starter-file{alg2-fuel-efficiency} open on your computer. If you haven't already, select **Save a Copy** from the "File" menu to make a copy of the file that's just for you.
+
+@n Click "Run", and evaluate `mpg-table` in the Interactions Area. Do you know what each column means?
+
+[cols="1a,1a", grid="none", frame="none", stripes="none"]
+|===
+|
+- *test* - the number of the fuel efficiency test
+- *model* - the car used in the test (A-F)
+|
+- *speed* - the speed at which the car was tested
+- *mpg* - the miles-per-gallon the car achieved at that speed
+|===
+
+@n Evaluate `A15`, `A45` and `A75` in the Interactions Area. What *model* of car is used in all three rows? @fitb{}{@ifsoln{`A`}}
+
+@n At what three *speeds* is this model being tested in these rows? @fitb{}{@ifsoln{15, 45 and 75 miles per gallon}}
+
+@vspace{1ex}
+
+[.FillVerticalSpace, cols="1a,1a", options="frame"]
+|===
+|
+@n In Pyret, make a scatter plot showing the relationship between `speed` and `mpg`, using `model` as your labels, then sketch the resulting scatter plot below. Be sure to label your axes!
+
+@ifsoln{@image{../images/fuel-efficiency-quadratic.png}}
+
+|
+Based on the scatter plot... +
+
+@n What do you *Notice?* @fitb{}{} +
+@fitb{}{} +
+@fitb{}{}
+
+@n What do you *Wonder?* @fitb{}{} +
+@fitb{}{} +
+@fitb{}{}
+
+@n Which model is generally the most fuel-efficient? @fitb{}{@ifsoln{B}}
```

Comment:
I'm not coming up with this answer, which makes me wonder how you can be sure it's THE right answer.
I think it might be better to ask them to "Identify a model that appears to be fuel-efficient" B, H, I, J...

Reply:
Adjusted.